akka-remote/src/test/scala/akka/remote/artery/ArteryMultiNodeSpec.scala

```diff
@@ -38,6 +40,15 @@ abstract class ArteryMultiNodeSpec(config: Config)
 
   private var remoteSystems: Vector[ActorSystem] = Vector.empty
 
+  override protected def withFixture(test: NoArgTest): Outcome = {
+    // note that withFixture is also used in FlightRecorderSpecIntegration
```

Comment:
I have verified that this works, invoked in right order, and still emits flight recorder output when test fails, but nothing when Pending.

I also tried a few other things like throwing from constructor via `pending`, but that didn't work.